upstream/jobs.go

```diff
@@ -0,0 +1,55 @@
+package upstream
+
+import (
+	"fmt"
+
+	"github.com/flanksource/commons/logger"
+	"github.com/flanksource/duty"
+	"github.com/flanksource/duty/models"
+	"github.com/google/uuid"
+)
+
+// SyncCheckStatuses pushes check statuses, that haven't already been pushed, to upstream.
+func SyncCheckStatuses(ctx duty.DBContext, config UpstreamConfig, batchSize int) error {
+	var checkStatuses []models.CheckStatus
```

Comment:
I think the the batching should occur here - There could be potentially tens of thousands of statuses to replicate.